service/slack-service.js

```diff
@@ -1,21 +1,150 @@
-const WebClient = require('@slack/client').WebClient;
 const config = require('../conf/config')
 const Promise = require("bluebird");
+const rp = require('request-promise');
+
 export class SlackService {
-    sendUserFeedBack(message) {
-        let bot_token = process.env.SLACK_BOT_TOKEN || config.slack.bottoken;
-        let feedbackChannel = config.slack.feedback_channel;
+
+    getBaseUrl(){
+        var host = config.slackserver.host;
+        var port = config.slackserver.port;
+
+        return host + ':' + port;
+    }
+    //post and get from a poc
+    postFeedbackToPoc(message) {
+        var url = this.getBaseUrl() + '/posttopoc';
+        var options = {
+            method: 'POST',
+            uri: url,
+            body: {
+                name: message.name,
+                location: message.location,
+                phone: message.phone,
+                message: message.message   
+            },
+            json: true 
+        };
+        
         return new Promise(function (resolve, reject) {
-            let web = new WebClient(bot_token);
-            web.chat.postMessage('G4GFP8BPU', message, function (err, res) {
-                if (err) {
-                    console.log('Error:', err);
-                    resolve({ status: err });
-                } else {
-                    console.log('Message sent: ', res);
-                    resolve({ status: 'okay' });
-                }
+            rp(options)
+            .then(function(parsedBody) {
+                console.log('Message sent: ',parsedBody );
```

Comment:
comment out this line, as it will appear in log files. This is commented out, and left for developer to uncomment during debugging